authenticator/components/application-list-icon.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { StyleSheet, Image, View } from 'react-native';
+
+export default function ApplicationListIcon( id ) {
+    return(
+        <View>
+            <Image style={styles.logo} source={{uri: 'https://reactnative.dev/img/tiny_logo.png'}}/>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
+    logo: {
+      width: 75,
+      height: 75,

```

Comment:
can we set the width and height as an input for the Application list view? If we want to use an icon or allow for better readability that would make it super easy